data/registers/rcc_c0.yaml

```diff
@@ -1,931 +1,2386 @@
+    fieldset: RCC_IOPSMENR
+    name: RCC_IOPSMENR
+  - byte_offset: 72
+    description: RCC AHB peripheral clock enable in Sleep/Stop mode register.
     fieldset: AHBSMENR
-  - name: APBSMENR1
-    description: RCC APB peripheral clock enable in Sleep/Stop mode register 1
-    byte_offset: 76
+    name: AHBSMENR
+  - byte_offset: 72
+    description: RCC AHB peripheral clock enable in Sleep/Stop mode register.
+    fieldset: RCC_AHBSMENR
+    name: RCC_AHBSMENR
+  - byte_offset: 76
+    description: RCC APB peripheral clock enable in Sleep/Stop mode register 1.
     fieldset: APBSMENR1
-  - name: APBSMENR2
-    description: RCC APB peripheral clock enable in Sleep/Stop mode register 2
-    byte_offset: 80
+    name: APBSMENR1
+  - byte_offset: 76
+    description: RCC APB peripheral clock enable in Sleep/Stop mode register 1.
+    fieldset: RCC_APBSMENR1
+    name: RCC_APBSMENR1
+  - byte_offset: 80
+    description: RCC APB peripheral clock enable in Sleep/Stop mode register 2.
     fieldset: APBSMENR2
-  - name: CCIPR
-    description: RCC peripherals independent clock configuration register
-    byte_offset: 84
+    name: APBSMENR2
+  - byte_offset: 80
+    description: RCC APB peripheral clock enable in Sleep/Stop mode register 2.
+    fieldset: RCC_APBSMENR2
+    name: RCC_APBSMENR2
+  - byte_offset: 84
+    description: RCC peripherals independent clock configuration register.
     fieldset: CCIPR
-  - name: CSR1
-    description: RCC control/status register 1
-    byte_offset: 92
+    name: CCIPR
+  - byte_offset: 84
+    description: RCC peripherals independent clock configuration register 1.
+    fieldset: RCC_CCIPR1
+    name: RCC_CCIPR1
+  - byte_offset: 88
+    description: RCC peripherals independent clock configuration register 2.
+    fieldset: RCC_CCIPR2
+    name: RCC_CCIPR2
+  - byte_offset: 92
+    description: RCC control/status register 1.
     fieldset: CSR1
-  - name: CSR2
-    description: RCC control/status register 2
-    byte_offset: 96
+    name: CSR1
+  - byte_offset: 92
+    description: RCC control/status register 1.
+    fieldset: RCC_CSR1
+    name: RCC_CSR1
+  - byte_offset: 96
+    description: RCC control/status register 2.
     fieldset: CSR2
+    name: CSR2
+  - byte_offset: 96
+    description: RCC control/status register 2.
+    fieldset: RCC_CSR2
+    name: RCC_CSR2
+enum/ADCSEL:
+  bit_size: 2
+  variants:
+  - description: System clock.
+    name: B_0x0
+    value: 0
+  - description: HSIKER.
+    name: B_0x2
+    value: 2
+enum/CSSC:
+  bit_size: 1
+  variants:
+  - description: No effect.
+    name: B_0x0
+    value: 0
+  - description: Clear CSSF flag.
+    name: B_0x1
+    value: 1
+enum/CSSF:
+  bit_size: 1
+  variants:
+  - description: Interrupt not pending.
+    name: B_0x0
+    value: 0
+  - description: Interrupt pending.
+    name: B_0x1
+    value: 1
+enum/HPRE:
+  bit_size: 4
+  variants:
+  - description: '2.'
+    name: B_0x8
```

Comment:
The format of this line appears to be one of the issues resulting in the error in `parse_num` - I guess this is normally like `Mul2` or something?